discord_connector.py

```diff
@@ -289,42 +348,48 @@ def handle_action(self, param):
 
         self.debug_print("action_id", self.get_action_identifier())
 
-        if action_id == 'list_gulds':
-            ret_val = self._handle_list_gulds(param)
+        if action_id == 'list_guilds':
+            ret_val = self._handle_list_guilds(param)
 
         if action_id == 'list_channels':
             ret_val = self._handle_list_channels(param)
 
+        if action_id == 'fetch_message':
+            ret_val = self._handle_fetch_message(param)
+
         if action_id == 'test_connectivity':
             ret_val = self._handle_test_connectivity(param)
 
         return ret_val
 
     def initialize(self):
         # Load the state in initialize, use it to store data
-        # that needs to be accessed across actions
         self._state = self.load_state()
 
         # get the asset config
         config = self.get_config()
-        """
-        # Access values in asset config by the name
-
-        # Required values can be accessed directly
-        required_config_name = config['required_config_name']
-
-        # Optional values should use the .get() function
-        optional_config_name = config.get('optional_config_name')
-        """
 
         self._base_url = "https://discord.com/api/v10"
         self._token = config['token']
-        self._headers={"Authorization": "Bot " + self._token}
+        self._guild_id = config['guild_id']
+
+        # obsolete:
```

Comment:
obsolete?

Reply:
It looks like a useful dict for reuse in many places.

Reply:
As we are using discord.py lib this is only going to be used in test_connectivity action (with we probably should update to test also if guild_id parameter is good)